tests/test-cases/Group6-VIC-Machine/6-07-Create-Network.robot

```diff
@@ -246,15 +245,15 @@ Bridge network - ESX none
     Cleanup VIC Appliance On Test Server
 
 Bridge network - invalid
-    Pass execution  asdf
+    Run Keyword If  '%{HOST_TYPE}' != 'ESXi'  Pass Execution  Test skipped on vCenter
```

Comment:
I took this from one of the other tests. Looking at the suite, there doesn't seem to be any consensus :p